prevent-large-objects.sh

```diff
@@ -12,6 +12,10 @@ max_size="${1:?need max object size}"
 
 found="false"
 while read -r obj_id path; do
+  if [ -z "$path" ]; then
+    # Skip objects which are not files (commits, trees)
```

Comment:
There should be no assumption that the commit is the one currently checked out.

Nor is there any reason to limit it to not-commits.
e.g. someone might be really silly and create a multi-megabyte commit message.